packages/@react-spectrum/combobox/src/ComboBox.tsx

```diff
@@ -203,6 +204,7 @@ const ComboBoxInput = React.forwardRef(function ComboBoxInput(props: ComboBoxInp
     autoFocus,
     style,
     UNSAFE_className,
+    className,
```

Comment:
noticed that this className is provided by `Field` and I had removed it by mistake in a prior pull so readded